telegram/chatmember.py

```diff
@@ -251,3 +268,341 @@ def to_dict(self) -> JSONDict:
         data['until_date'] = to_timestamp(self.until_date)
 
         return data
+
+
+class ChatMemberOwner(ChatMember):
+    """
+    Represents a chat member that owns the chat
+    and has all administrator privileges.
+
+    .. versionadded:: 13.7
+
+    Args:
+        status (:obj:`str`): The member's status in the chat,
+            always “creator”.
+        user (:class:`telegram.User`): Information about the user.
+        custom_title (:obj:`str`, optional): Custom title for this user.
+        is_anonymous (:obj:`bool`, optional):  :obj:`True`, if the
+            user's presence in the chat is hidden.
+
+    Attributes:
+        status (:obj:`str`): The member's status in the chat,
+            always “creator”.
+        user (:class:`telegram.User`): Information about the user.
+        custom_title (:obj:`str`): Optional. Custom title for
+            this user.
+        is_anonymous (:obj:`bool`): Optional. :obj:`True`, if the user's
+            presence in the chat is hidden.
+    """
+
+    __slots__ = ()
+
+    def __init__(
+        self,
+        status: str,
```

Comment:
status *must* be `'creator'` here, so I'd vote to skip this argument and just pass the right value to super (like we do for e.g. `InlineQueryResult*`). Same for the other classes.